cli/cmd/inject.go

```diff
@@ -28,8 +28,8 @@ const (
 	injectDisabledDesc               = "pods are not annotated to disable injection"
 	unsupportedDesc                  = "at least one resource injected"
 	udpDesc                          = "pod specs do not include UDP ports"
+	automountServiceAccountTokenDesc = "pods are allowed to automount API credentials"
```

Comment:
I would prefer to be more precise here with the message mentioning service account tokens are automounted, instead of talking about API credentials.